### PR TITLE
[Serverless] Fix CVE-2022-27664

### PR DESCRIPTION
**Pull request by maxday**
### What does this PR do?

Bump dependencies to fix CVE-2022-27664

### Motivation

CVE-2022-27664

### Describe how to test/QA your changes

Unit + integration tests

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by ajgajg1134 — APPROVED** [review]
👍 for agent-apm

File: test/integration/serverless/recorder-extension/go.mod [hunk 1]
```diff
@@ -36,4 +36,5 @@ replace (
 require (
 	github.com/DataDog/agent-payload/v5 v5.0.19
 	github.com/DataDog/datadog-agent v0.0.0-20211213161047-f82981e22ca1
+	golang.org/x/net v0.1.0 // indirect
```
Comment (agocs) [inline test/integration/serverless/recorder-extension/go.mod:39]: weird that we're adding an indirect dependency here and in that other place. I assume `go mod` did that for you?

**Review by agocs — APPROVED** [review]
LGTM

File: test/integration/serverless/recorder-extension/go.mod (hunk 1 above)
Comment (maxday) [reply test/integration/serverless/recorder-extension/go.mod:39]: yes, also this won't reach the binary that we deliver but the static analysis tool was complaining

**Review by kimi-p — APPROVED** [review]

**Pull request merged**